x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/trial_license_complete_tier/import_rules.ts

```diff
@@ -770,435 +356,205 @@ export default ({ getService }: FtrProviderContext): void => {
-            .post(`${DETECTION_ENGINE_RULES_IMPORT_URL}`)
-            .set('kbn-xsrf', 'true')
-            .set('elastic-api-version', '2023-10-31')
-            .attach('file', buffer, 'rules.ndjson')
-            .expect(200);
+        expect(exceptionListItem).toMatchObject(expected);
+      };
 
-          const { body } = await supertest
-            .post(
-              `/s/${spaceId}${DETECTION_ENGINE_RULES_IMPORT_URL}?overwrite=true&overwrite_action_connectors=true`
-            )
-            .set('kbn-xsrf', 'true')
-            .set('elastic-api-version', '2023-10-31')
-            .attach('file', buffer, 'rules.ndjson')
-            .expect(200);
+      describe('rules with exceptions', () => {
+        it('imports a rule with a single space exception', async () => {
+          const CUSTOM_QUERY_RULE_WITH_EXCEPTION = getCustomQueryRuleParams({
+            rule_id: RULE_TO_IMPORT_RULE_ID,
+            exceptions_list: [
+              {
+                id: 'single',
+                list_id: 'test_list_id',
+                type: 'rule_default',
+                namespace_type: 'single',
+              },
+            ],
+          });
+          const EXCEPTION_LIST = {
+            ...getImportExceptionsListSchemaMock('test_list_id'),
+            type: 'rule_default',
+          };
+          const EXCEPTION_LIST_ITEM = getImportExceptionsListItemNewerVersionSchemaMock(
+            'test_item_id',
+            'test_list_id'
+          );
+          const IMPORT_PAYLOAD = [
+            CUSTOM_QUERY_RULE_WITH_EXCEPTION,
+            EXCEPTION_LIST,
+            EXCEPTION_LIST_ITEM,
+          ];
+
+          const importResponse = await importRules({
+            getService,
+            rules: IMPORT_PAYLOAD,
+            overwrite: false,
+            spaceId: kibanaSpaceId,
+          });
 
-          expect(body).toMatchObject({
+          expect(importResponse).toMatchObject({
             success: true,
             success_count: 1,
             rules_count: 1,
             errors: [],
-            action_connectors_success: true,
-            action_connectors_success_count: 1,
-            action_connectors_warnings: [],
-            action_connectors_errors: [],
+            exceptions_errors: [],
+            exceptions_success: true,
+            exceptions_success_count: 1,
           });
-        });
-      });
 
-      describe('@skipInServerless migrate pre-8.0 action connector ids', () => {
-        const defaultSpaceActionConnectorId = '61b17790-544e-11ec-a349-11361cc441c4';
-        const space714ActionConnectorId = '51b17790-544e-11ec-a349-11361cc441c4';
-
-        beforeEach(async () => {
-          await esArchiver.load(
-            'x-pack/test/functional/es_archives/security_solution/import_rule_connector'
-          );
-        });
-        afterEach(async () => {
-          await esArchiver.unload(
-            'x-pack/test/functional/es_archives/security_solution/import_rule_connector'
```

Comment:
Can this archive `import_rule_connector`, and its reference in `src/platform/packages/shared/kbn-es-archiver/src/fixtures/override_saved_objects_index/exception_list.json`, now be deleted? That seems to be the only remaining reference.